app/views/direct_assessments/show.html.erb

```diff
@@ -1,25 +1,43 @@
 <h1><%= "#{@assessment.name} - #{@assessment.description}" %></h1>
 <h2><%= @assessment.subject %></h2>
+<ul>
+  <li>Problem description: <%= @assessment.problem_description %></li>
+  <li>Minimum grade: <%= @assessment.minimum_requirement %></li>
+  <li>Target percentage: <%= @assessment.target_percentage %></li>
+</ul>
 
-<table>
-  <tr>
-    <th>Name</th>
-    <th>Description</th>
-    <th>Problem Description</th>
-    <th>Year</th>
-    <th>Semester</th>
-    <th>Percentage</th>
-  </tr>
-  <% @assessment.results.each do |result| %>
+<% if @assessment.results.any? %>
```

Comment:
I think that this should be `present?`. We're going to iterate it if there are any, which means we'd need to go back to the database to get all the records. What do you think?